pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -2250,8 +2253,14 @@ func handleGlobalConfigImpl(ctxArg interface{}, key string,
 			updateVgaAccess(ctx)
 			ctx.setInitialVgaAccess = true
 		}
-		if gcp.GlobalValueInt(types.MetricInterval) != 0 {
-			ctx.metricInterval = gcp.GlobalValueInt(types.MetricInterval)
+		metricInterval := gcp.GlobalValueInt(types.MetricInterval)
+		if metricInterval != 0 && ctx.metricInterval != metricInterval {
+			// adjust publishTicker interval if metricInterval changed
+			max := float64(time.Duration(metricInterval) * time.Second)
```

Comment:
I think we want the max to be some factor less than metricsInterval so that the data that is sent to the controller isn't too stale.

If metricsInterval is set to 10 minutes that means there can be a 20 minute delay from some event (which incremented the metrics) because domainmgr might publish 10 minutes later and then zedagent might publish another 10 minutes after than.

This is why we have the (arbitrary) 6x ratio today between the default 60 seconds and the 10 seconds here.